x-pack/plugins/fleet/common/services/generate_new_agent_policy.test.ts

```diff
@@ -17,7 +17,6 @@ describe('generateNewAgentPolicyWithDefaults', () => {
       namespace: 'default',
       monitoring_enabled: ['logs', 'metrics'],
       inactivity_timeout: 1209600,
-      is_protected: false,
```

Comment:
why are we removing this?

Reply:
We want to take this out since `NewPackagePolicy` doesn't have `package_policies`, and therefore the defend integration, during creation, and to prevent users from modifying this field. Even if a user were to add the defend integration to a new agent policy, as opposed to an existing one, the new agent policy is created first before adding the defend package policy, and then at this point, the `is_protected` field would be available but defaulted to false.